src/HealthGPS.Tests/AgeGenderTable.Test.cpp

```diff
@@ -159,7 +159,7 @@ TEST(TestHealthGPS_AgeGenderTable, CreateWithWrongRangerThrows) {
     using namespace hgps;
 
     auto negative_range = core::IntegerInterval(-1, 10);
-    auto inverted_range = core::IntegerInterval(10, 1);
+    auto inverted_range = core::IntegerInterval(1, 1);
```

Comment:
As you've dropped support for inverted ranges, this bit of the test can be dropped.

Reply:
That particular bit is testing for failure with zero-width intervals. The age_range is [lo, hi) -- i.e. low-inclusive, high-exclusive.

Reply:
Ahh I seeeeee. That makes sense